charts/kamaji-etcd/templates/etcd_datastore.yaml

```diff
@@ -3,9 +3,6 @@ apiVersion: kamaji.clastix.io/v1alpha1
 kind: DataStore
 metadata:
   name: {{ include "etcd.fullname" . }}
-  annotations:
-    "helm.sh/hook": post-install,post-upgrade,post-rollback
-    "helm.sh/hook-weight": "5"
```

Comment:
To avoid the Datastore from being delete upon every upgrade, or rollback, the hooks must be removed.